lib/ansible/modules/cloud/digital_ocean/_digital_ocean.py

```diff
@@ -1,487 +0,0 @@
-#!/usr/bin/python
```

Comment:
Why is this removed? It is not supposed to be removed until 2.12.

Reply:
True, missed this one

Reply:
I missed that.

Reply:
Fixed